README.md

```diff
@@ -43,39 +56,84 @@ The following guide is for members of the project team who have access to the re
 
 ## Local Development
 
-<!--- TODO - with example below:
-This project is monorepo with several apps. Please see the [api](./api/README.md) and [frontend](./frontend/README.md) READMEs for information on spinning up those projects locally. Also see the project [documentation](./documentation) for more info.
--->
+
+### Installation
+
+1. Clone the repo
+
+    `git clone https://github.com/IsaacMilarky/gource-log.git`
+
+2. Install the required packages and build dependencies:
+
+    - Docker
+    - act
+    - gource
+    
+## Running Locally
+
+1. Change to the base project directory
+
+2. Run the 'build' job using act
+
+    `sudo act -a build`
 
 ## Coding Style and Linters
 
-<!-- TODO - Add the repo's linting and code style guidelines -->
 
-Each application has its own linting and testing guidelines. Lint and code tests are run on each commit, so linters and tests should be run locally before committing.
+This project adheres to PEP8 rules and guidelines whenever possible when accepting
+new contributions of Python code. Although, there are good reasons to ignore particular guidelines
+in particular situations. Further information on PEP8 can be found [here.](https://peps.python.org/pep-0008/)
+
+This project utilizes pylint as the primary linter for backend code, while eslint and prettier handle code formatting and linting for the frontend. Checks are implemented upon new pull requests into protected branches to ensure code quality and consistency.
+
+Python code quality checks are extremely useful for lowering the
+cost of maintenence of Python projects. Further information on Pylint can be found [here.](https://pylint.readthedocs.io/en/latest/)
+
+## Coding Style and Linters
```

Comment:
"Code Style and Linters" is on here twice

Reply:
Whoops will fix